linkerd/app/src/lib.rs

```diff
@@ -160,18 +160,15 @@ impl Config {
         let inbound = {
             let inbound = inbound;
             let identity = identity.local();
-            // let profiles = dst.profiles.clone();
+            let profiles = dst.profiles.clone();
             // let tap = tap.layer();
             let metrics = metrics.inbound;
             // let oc = oc_collector.span_sink();
             let drain = drain_rx.clone();
             info_span!("inbound").in_scope(move || {
                 inbound.build(
-                    identity,
-                    (), // profiles,
-                    // tap,
-                    metrics,
-                    None, // oc,
+                    identity, profiles, // tap,
+                    metrics, None, // oc,
```

Comment:
This is odd formatting. This should change though as tap and oc are added back in

Reply:
Yup, rustfmt is dead-set on formatting it this way, as far as I can tell, but hopefully it will be a bit less bad when those come back.